catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -135,6 +146,46 @@ public Metacard getMetacard(String id)
         return result.getMetacard();
     }
 
+    /**
+     * Returns all associated metacards given two relatable attributes.
+     *
+     * @param primaryFilterAttribute   The primary attribute to search against
+     * @param secondaryFilterAttribute The secondary attribute to search against
+     * @param primaryAttributeValue    A primary attribute value
+     * @param secondaryAttributeValue  A secondary attribute value
+     * @return A set of metacards, filtered by the given attributes and it's values
+     * @throws UnsupportedQueryException  Error
+     * @throws SourceUnavailableException Error
+     * @throws FederationException        Error
+     */
+    public List<Metacard> getAssociatedMetacardsByTwoAttributes(String primaryFilterAttribute,
+            String secondaryFilterAttribute, String primaryAttributeValue,
+            String secondaryAttributeValue)
+            throws UnsupportedQueryException, SourceUnavailableException, FederationException {
+        LinkedList<Metacard> resultList = new LinkedList<>();
+        Filter idFilter = filterBuilder.attribute(primaryFilterAttribute)
+                .is()
+                .equalTo()
+                .text(primaryAttributeValue);
+        Filter tagsFilter = filterBuilder.attribute(secondaryFilterAttribute)
+                .is()
+                .equalTo()
+                .text(secondaryAttributeValue);
+        Filter filter = filterBuilder.allOf(idFilter, tagsFilter);
+        QueryResponse queryResponse = catalogFramework.query(new QueryRequestImpl(new QueryImpl(
+                filter), false));
+        List<Result> results = queryResponse.getResults();
+        results.forEach(resultObject -> resultList.add(resultObject.getMetacard()));
+        return resultList;
+    }
+
+    public String getResponseWrapper(String responseType, Object response) {
+        JSONObject responseObj = new JSONObject();
```

Comment:
❗️ We should not be using `org.json.JSONObject`here since we already use boon throughout this module.

Reply:
switched to boon